v2/klipper/macros/printing.cfg

```
@@ -85,7 +89,7 @@ gcode:
       _NOTIFY T="Calibrating z-offset"
       G1 Z{config.travel_clearance} F{config.travel_speed}
       G1 X{printer.toolhead.axis_maximum.x / 2} Y{printer.toolhead.axis_maximum.y / 2} F{config.travel_speed}
-      TEMPERATURE_WAIT SENSOR=extruder MINIMUM={probe_temp} MAXIUMUM={probe_temp+5}
+      TEMPERATURE_WAIT SENSOR=extruder MINIMUM={probe_temp} MAXIMUM={probe_temp+5}
```

We should make this match our general MPC temp ranges

```suggestion
      TEMPERATURE_WAIT SENSOR=extruder MINIMUM={probe_temp-2} MAXIMUM={probe_temp+5}
```